h2o-docs/src/product/security.rst

```diff
@@ -782,9 +782,9 @@ tool:
 
     java -cp h2o.jar org.eclipse.jetty.util.security.Password username password
 
-See the `Jetty 8 HashLoginService
+See the `Jetty 9 HashLoginService
 documentation <http://wiki.eclipse.org/Jetty/Tutorial/Realms#HashLoginService>`_
-and `Jetty 8 Secure Password
+and `Jetty 9 Secure Password
 HOWTO <http://wiki.eclipse.org/Jetty/Howto/Secure_Passwords>`_ for more
```

Comment:
The "HOWTO" link is now a redirect. I think the link for the Secure Password should be changed to:
http://www.eclipse.org/jetty/documentation/current/configuring-security-secure-passwords.html

Reply:
Good point @angela0xdata, thanks!